moler/cmd/commandtextualgeneric.py

```diff
@@ -285,6 +285,12 @@ def is_end_of_cmd_output(self, line):
         """
         if self._regex_helper.search_compiled(self._re_prompt, line):
             return True
+        # when command is broken via Ctrl-C then ^C may be appended to start of prompt
+        # if prompt regexp requires "at start of line" via r'^' then such ^C concatenation will falsify prompt
+        if (len(line) > 2) and line.startswith("^C"):
```

Comment:
What do you think to add flag like self._remove_ctrlc_chars_for_prompt (default value: True)  and put this code into condition for this flag? If any command doesn't want to remove ctrl+c chars then it can assign False to the flag.

Reply:
Agree